org.civicrm.bootstrapcivihr/scss/components/outside-namespace/_h-scroll-shadows.scss

```diff
@@ -0,0 +1,42 @@
+.chr_h-scroll-shadows__master {
+  overflow: hidden; // this ensures the shadows are hidden behind
+  position: relative;
+}
+
+.chr_h-scroll-shadows__wrapper {
+  overflow-x: auto;
+
+  &::before,
+  &::after {
+    box-shadow: 0 0 10px 1px rgba(48, 40, 40, 0.5);
+    content: '';
+    display: block;
+    height: calc(100% - 10px); // if height is 100% then the shadow will be cut off
+    opacity: 0; // by default the shadows are invisible
+    position: absolute;
+    top: 5px;
+    transition: opacity 0.2s ease-out;
+    width: 1px; // in order to make the shadow appear
+    z-index: 1; // in order to appear on top of the inset content
+  }
+
+  &::before {
+    // This hides the shadow element underneath the master element
+    // leaving the actual shadow still visible
+    left: -1px;
+  }
+
+  &::after {
+    right: -1px;
+  }
+
+  &.chr_h-scroll-shadows__wrapper--inset-shadow-left::before,
+  &.chr_h-scroll-shadows__wrapper--inset-shadow-right::after {
+    opacity: 1;
+  }
+
+  .chr_h-scroll-shadows__content {
+    display: inline-block; // to appear with :after and :before properly
```

Comment:
what does "to appear with :after and :before properly" mean? what happens if you don't set `inline-block`?

Reply:
It will collapse the content.